src/pykeen/models/base.py

```diff
@@ -107,8 +102,9 @@ def __init__(
         else:
             self.loss = loss
 
-        # The triples factory facilitates access to the dataset.
-        self.triples_factory = triples_factory
+        self._create_inverse_triples = triples_factory.create_inverse_triples
```

Comment:
Since we do not store the triples factory anymore, we can directly store this into an attribute: `self.num_entities`

Reply:
Done in eac8cc47